server/views/components/font-example/font-example.njk

```diff
@@ -0,0 +1,14 @@
+<div class="font">
+  <h2 class="font__name">{{ font.font__name }}</h2>
+  <p class="font__example--{{ font.font__name }}">{{ font.font__example }}</p>
+  <dl class="font__properties">
+    <dt class="font__properties__property">Font-size:</dt>
+    <dd class="font__properties__value">{{ font.font__size }}</dd>
+    <dt class="font__properties__property">Letter-spacing:</dt>
+    <dd class="font__properties__value">{{ font.letter__spacing }}</dd>
+    <dt class="font__properties__property">Line-height:</dt>
+    <dd class="font__properties__value">{{ font.line__height }}</dd>
```

Comment:
Will change in line with David's comments.

-- should be reserved for modifiers though, not elements